### PR TITLE
PUBDEV-8558: Example of combining multiple POJOs + feature engineering

### PR DESCRIPTION
**Pull request by michalkurka**

### CODE REVIEW

**Review by neemah2o — CHANGES_REQUESTED** [review]
Looks good other than add info on %s() , line 84

File: h2o-py/tests/testdir_generic_model/pyunit_pojo_import.py [hunk 1]
```diff
@@ -0,0 +1,334 @@
...
+    private final GenModel glm;
+    private final GenModel gbm;
+
+    // for each sub-model, mapping of the main model input and of the calculated columns to the sub-model input
+    private final Map<String, int[]> mappings;
+
+    // map of feature names to feature indices in the input array
+    private final Map<String, Integer> featureMap;
+
+    public %s() {
```
[the remark this answers is not in the document]
Comment (michalkurka) [reply h2o-py/tests/testdir_generic_model/pyunit_pojo_import.py:84]: thanks, will add documentation to the methods. This is a constructor that initializes the class and all the inner fields.

**Comment by michalkurka** [comment]
@neemah2o please take another, look - is it clearer now?

**Comment by neemah2o** [comment]
Yes, it's more clear now

**Review by neemah2o — APPROVED** [review]

**Pull request merged**